src/actions/reported.actions.js

```diff
@@ -69,10 +83,31 @@ const reportArticle = (articleid, body) => {
   };
 };
 
+const reviewArticle = (articleid, commentBody, reportid) => {
+  return async dispatch => {
+    dispatch(contentLoading());
+    try {
+      const res = await http.patch(`/article/review/${articleid}`, commentBody);
+      if (res.status === 200) {
```

Comment:
Please why are you doing this here? I think the try block is suppose to handle success